internal/printer/pod.go

```diff
@@ -70,6 +70,22 @@ func PodListHandler(ctx context.Context, list *corev1.PodList, opts Options) (co
 
 		row["Phase"] = component.NewText(string(pod.Status.Phase))
 
+		if len(pod.Status.ContainerStatuses) > 0 {
+			lastStatus := pod.Status.ContainerStatuses[len(pod.Status.ContainerStatuses)-1]
```

Comment:
Nitpicking here but the logic to determine the status can go inside a private method.

Reply:
Could be but don't you think that the logic is too simple for that and also a method would have at least 3 lines of code and here we can have it with just one